styx-common/src/test/java/com/spotify/styx/state/RunStateTest.java

```diff
@@ -101,6 +101,7 @@ public void testRunErrorOnCreating() throws Exception {
   public void testSetExecutionId() throws Exception {
     transitioner.initialize(RunState.fresh(WORKFLOW_INSTANCE, this::record));
     transitioner.receive(eventFactory.triggerExecution("trig"));
+    transitioner.receive(eventFactory.dequeue());
```

Comment:
I think these tests should not be modified to add a `dequeue` event but instead the state machine has to be backwards compatible with `triggerExecution -> created` sequences. See comment above.